src/test/kotlin/uk/gov/dluhc/printapi/messaging/ProcessPrintRequestBatchMessageListenerIntegrationTest.kt

```diff
@@ -54,28 +62,118 @@ internal class ProcessPrintRequestBatchMessageListenerIntegrationTest : Integrat
             id = printDetailsId,
             batchId = batchId,
             printRequestStatuses = mutableListOf(
-                PrintRequestStatus(ASSIGNED_TO_BATCH, OffsetDateTime.now(clock))
+                PrintRequestStatus(ASSIGNED_TO_BATCH, OffsetDateTime.now(clock), OffsetDateTime.now(clock))
             ),
             requestId = requestId,
             eroWelsh = buildElectoralRegistrationOffice(),
             photoLocation = "arn:aws:s3:::$s3Bucket/$s3Path"
         )
         printDetailsRepository.save(details)
+
+        // save certificates in MySQL
+        var certificate = Certificate(
```

Comment:
Verified manually that the only difference in the generated File Contents is difference in timestamp precision.
This test will be simplified after DynamoDB code is removed.